bindings_ffi/src/mls.rs

```diff
@@ -4129,7 +4129,7 @@ mod tests {
         message_callbacks.wait_for_delivery(None).await.unwrap();
         assert_eq!(bo.provider.db().intents_published(), 4);
 
-        assert_eq!(message_callbacks.message_count(), 5);
+        assert_eq!(message_callbacks.message_count(), 6);
```

Comment:
This test fails for me on `main` as well. 6 seems like the right number, not 5. Not sure how we've been getting through CI with 5.